openverse_catalog/dags/providers/provider_api_scripts/wordpress.py

```diff
@@ -229,7 +229,6 @@ def _extract_image_data(media_data):
     image_url, height, width, filetype = _get_file_info(image_details)
     if image_url is None:
         return None
-    thumbnail = _get_thumbnail_url(image_details)
```

Comment:
It looks like we can delete the `_get_thumbnail_url` method as well.

Reply:
Thank you for catching this :)